apps/nova/src/app/[locale]/(dashboard)/challenges/[challengeId]/results/page.tsx

```
@@ -10,6 +16,7 @@
 import { Button } from '@tuturuuu/ui/button';
 import { ArrowLeft, BookOpen, Loader2, X } from 'lucide-react';
 import { useRouter } from 'next/navigation';
+import { redirect } from 'next/navigation';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'redirect'. Consider to delete it or rename it.

'redirect' is defined here:


(lint/suspicious/noRedeclare)